tests/api.rs

```diff
@@ -50,6 +62,28 @@ mod tests {
         .await
     }
 
+    async fn query<T: GraphQLQuery>(
+        request_body: &graphql_client::QueryBody<T::Variables>,
+    ) -> graphql_client::Response<T::ResponseData> {
+        let config = api_enabled_config();
+        let addr = config.api.bind.unwrap();
+        let url = format!("http://{}:{}/graphql", addr.ip(), addr.port());
+
+        start_api_server(addr, false);
+
+        let client = reqwest::Client::new();
+
+        retry_until(
+            || client.post(&url).json(&request_body).send(),
```

Comment:
GraphQL queries could be delivered over any transport (even none at all, if we're using `schema.execute` locally/directly.)